### ISSUE TITLE
tail should support "+" correctly

### ISSUE THREAD

**Issue by nfischer** [shelljs/shelljs#984]
### Node version (or tell us if you're using electron or some other framework):

all

### ShellJS version (the most recent version/Github branch you see the bug on):

master

### Operating system:

all

### Description of the bug:

`shell.tail` doesn't support `+<num>` correctly. While it's true negative numbers should be treated the same as sign-less numbers (ex. `tail -n -10` means "print last 10 lines of file"), numbers with a `+` sign mean to print that numbered line and all lines following. So `tail -n +2 file.txt` means "print all lines except the first line."

### Example ShellJS command to reproduce the error:

```sh
$ shx tail -n +10 file_with_11_lines.txt
2
3
4
5
6
7
8
9
10
11

$ tail -n +10 file_with_11_lines.txt
10
11
```

**Comment by A-725-K** [comment]
Hello ! I would like to inform You (@nfischer) that I have addressed this issue in PR #1010. I have also added 2 more unit tests to verify my proposal (in test/tail.js), I hope I understand well the request. I would appreciate your feedback on my fix.  Thank you 😄

**Comment by nfischer** [comment]
Fixed by issue 1027
